community/index.html

```diff
@@ -421,6 +421,11 @@ <h1 class="h2">
                     <img class="contribute-topic-image" src="http://hoodiehq.github.io/hoodie-css/src/content_img/community/default-sw/jens-nikolaus.jpg" width="170" height="170" alt="Portrait photo(black and white) of Jens Nikolaus.">
                 </a>
             </div>
+            <div class="grid-3 l">
+                <a href="https://github.com/co16353sidak" target="_blank">
+                    <img class="contribute-topic-image" src="http://hoodiehq.github.io/hoodie-css/src/content_img/community/default-sw/sidak-singh.jpg" width="170" height="170" alt="Portrait photo(black and white) of Sidak Singh Aulakh.">
+                </a>
+            </div>
```

Comment:
I’m sorry Sidak, but we only added the core team to this list. Hoodie had far more than 100 contributors, the people listed here are only a subset, with a lot of contributions :) So please remove that change

Reply:
Let me revert this right away